quickstart/src/index.js

```diff
@@ -7,32 +7,50 @@ var previewTracks;
 var identity;
 var roomName;
 
-// Attach the Tracks to the DOM.
-function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    container.appendChild(track.attach());
-  });
+
+// Attach given track to the DOM.
+function attachTrack(container, track) {
+  container.appendChild(track.attach());
 }
 
-// Attach the Participant's Tracks to the DOM.
-function attachParticipantTracks(participant, container) {
-  var tracks = getTracks(participant);
-  attachTracks(tracks, container);
+// Attach array of Tracks to the DOM.
+function attachTracks(container, tracks) {
+  tracks.forEach(track => attachTrack(container, track));
 }
 
-// Detach the Tracks from the DOM.
-function detachTracks(tracks) {
-  tracks.forEach(function(track) {
-    track.detach().forEach(function(detachedElement) {
-      detachedElement.remove();
-    });
-  });
+// Detach given track from the DOM
+function detachTrack(track) {
+  track.detach().forEach(element => element.remove());
```

Comment:
@makarandp0 Can we use only es5 functions here? We can convert the quickstart app to es6 separately.